webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/FileSystemDataVault.scala

```diff
@@ -41,6 +42,10 @@ class FileSystemDataVault extends DataVault {
       }
     } else Fox.empty
 
+  override def hashCode(): Int =
+    new HashCodeBuilder(19, 31).toHashCode
+
+  override def equals(obj: Any): Boolean = true
```

Comment:
:trollface:

Reply:
note to self: this should still do the typecheck with `match`